### ISSUE TITLE
I have found that using the word segmentation method provided in your code, the token_id appears beyond the length of the dictionary

### ISSUE THREAD

**Issue by dshwei** [RUCKBReasoning/codes#10]
{'input_ids': tensor([[   58,    71, 14174,  ...,   642, 38775,  3641],
        [   61,    50,    54,  ...,    53,    56,    53],
        [ 1199,   490,   222,  ...,   648,    45,    58],
        [   55,    49,    57,  ...,    54,    60,    63],
        [   54,    50,    53,  ...,  4050,  1199,    49]]), 
        'attention_mask': tensor([[1., 1., 1.,  ..., 1., 1., 1.],
        [1., 1., 1.,  ..., 1., 1., 1.],
        [1., 1., 1.,  ..., 1., 1., 1.],
        [1., 1., 1.,  ..., 1., 1., 1.],
        [1., 1., 1.,  ..., 1., 1., 1.]]), 
        'labels': tensor([[   58,    71, 14174,  ...,   642, 38775,  3641],
        [   61,    50,    54,  ...,    53,    56,    53],
        [ 1199,   490,   222,  ...,   648,    45,    58],
        [   55,    49,    57,  ...,    54,    60,    63],
        [   54,    50,    53,  ...,  4050,  1199,    49]])}
        
        
        
such as  38775 id is out vocable size .
 how can this problem occur ? I can not find  the reason

**Comment by lihaoyang-ruc** [comment]
If you're referring to the input for CodeS, then yes, it is as expected. The reason is that CodeS has been designed with a vocabulary size of 49,152.
